cmake/adjust/flags.cmake

```diff
@@ -1,26 +1,58 @@
 include_guard(GLOBAL)
 
+include(CheckCCompilerFlag)
+include(CheckCXXCompilerFlag)
+
+if(NOT DEFINED _NBL_JOBS_AMOUNT_)
+	message(WARNING "\"${CMAKE_CURRENT_LIST_FILE}\" included without defined \"_NBL_JOBS_AMOUNT_\", setting it to \"1\"")
+	set(_NBL_JOBS_AMOUNT_ 1)
+endif()
+
 define_property(TARGET PROPERTY NBL_CONFIGURATION_MAP
   BRIEF_DOCS "Stores configuration map for a target, it will evaluate to the configuration it's mapped to"
 )
 
-macro(_NBL_IMPL_GET_FLAGS_PROFILE_)
-	if(MSVC)
-		include("${CMAKE_CURRENT_FUNCTION_LIST_DIR}/template/windows/msvc.cmake")
-	elseif(ANDROID)
-		include("${CMAKE_CURRENT_FUNCTION_LIST_DIR}/template/unix/android.cmake")
-	elseif(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-		include("${CMAKE_CURRENT_FUNCTION_LIST_DIR}/template/unix/gnu.cmake")
-	elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-		include("${CMAKE_CURRENT_FUNCTION_LIST_DIR}/template/unix/clang.cmake")
-	else()
-		message(WARNING "UNTESTED COMPILER DETECTED, EXPECT WRONG OPTIMIZATION FLAGS! SUBMIT ISSUE ON GITHUB https://github.com/Devsh-Graphics-Programming/Nabla/issues")
-	endif()
-endmacro()
+function(NBL_REQUEST_COMPILE_OPTION_SUPPORT _NBL_COMPILE_OPTION_)
+    set(NBL_COMPILE_OPTION "${_NBL_COMPILE_OPTION_}")
 
-function(NBL_EXT_P_APPEND_COMPILE_OPTIONS NBL_LIST_NAME MAP_RELEASE MAP_RELWITHDEBINFO MAP_DEBUG)
-	_NBL_IMPL_GET_FLAGS_PROFILE_()
-		
+    foreach(COMPILER IN ITEMS c cxx)
```

Comment:
why do I see my own changes as diff here? Did you pull @devshgraphicsprogramming or something?

Reply:
bad pull, solved